psi4/src/CMakeLists.txt

```diff
@@ -34,6 +34,13 @@ target_link_libraries(core PRIVATE ${psi4_binmodules})
 target_link_libraries(core PRIVATE ${LIBC_INTERJECT})
 target_link_libraries(core PUBLIC pybind11::module)
 target_link_libraries(core PRIVATE Threads::Threads)
+
+if(Fortran_ENABLED AND CMAKE_Fortran_COMPILER_ID MATCHES Intel)
```

Comment:
Actually, maybe not. It might be possible to compile the C++ source with intel and the fortran sources with something else. Which means that rtl_init, etc, are not required.

It should only be enabled if using the fortran compiler. So maybe a more descriptive flag would be better.